modules/katello_lifecycle_environment.py

```diff
@@ -75,7 +75,7 @@
   katello_lifecycle_environment:
     username: "admin"
     password: "changeme"
-    server_url: "https://katello.example.com"
+    server_url: "https://foreman.example.com"
```

Comment:
I started using `katello.example.com` to specifically indicate that the server must have the Katello plugin, not just Foreman. I'd vote to scratch these `server_url` changes and perhaps even go ahead and change the `server_url` on the other `katello_` modules to `katello.example.com`.